src/odata-to-abstract-sql.ts

```diff
@@ -122,6 +122,15 @@ const rewriteDefinition = (
 	return rewrittenDefinition;
 };
 
+const containsQueryOption = (opts: object): boolean => {
```

Comment:
Possibly, but the point of this function is to check if an object has a property starting with $, so it actually makes sense to have a loose typing imo since it's a type checker itself (eg like how `_.isString` is typed as `(x: any): x is boolean`)